spec/features/admin/admin_edits_user_spec.rb

```diff
@@ -20,8 +20,8 @@
     expect(page).
       to have_text(I18n.t(
         "flash.actions.update.notice",
-        resource_name: "User"
-      ))
+        resource_name: "User",
+      ),)
```

Comment:
Same ^

Reply:
Done in PR ad-refactor-flash-format-specs sha: 4be77c8